### PR TITLE
Fixes the bloodbeam

### PR DESCRIPTION
**Pull request by Sparkezel**
## About The Pull Request
Blood beam had wonky aiming for very long time, now its finally fixed.

## Why It's Good For The Game
Bugs and unintended aiming behaviour is bad 

## Changelog
:cl: SparkezelPL

fix: Blood beam now can be aimed properly once more!

/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
This has been a thing for nearly...two years? Really shows how little the blood rites spells are able to be used.

**Comment by Sparkezel** [comment]
Honestly it just needs few gamers to show the power of them, and the spells are quite robust. I think of it as specialist weapons, something only skilled few know how to use properly

**Comment by Sparkezel** [comment]
Honestly I am also thinking of buffing blood bolt a bit, so it can heal cultists similiarly to the beam but thats for later 😺

File: code/modules/antagonists/cult/cult_items.dm [hunk 1]
```diff
@@ -833,6 +833,8 @@
 	var/C = user.client
 	if(ishuman(user) && C)
 		angle = mouse_angle_from_client(C)
```
Comment (ATH1909) [inline code/modules/antagonists/cult/cult_items.dm:835]: does this line need to be here anymore?

**Comment by ATH1909** [comment]
> Honestly I am also thinking of buffing blood bolt a bit, so it can heal cultists similiarly to the beam but thats for later 😺

blood bolt already does that; it injects them with unholy water

**Comment by Sparkezel** [comment]
> > Honestly I am also thinking of buffing blood bolt a bit, so it can heal cultists similiarly to the beam but thats for later 😺
> 
> blood bolt already does that (or, well, is intended to do that)
Last I checked it still delt 20 damage to cultists, however that was on a catatonic mob so I am not sure.

File: code/modules/antagonists/cult/cult_items.dm (hunk 1 above)
Comment (Sparkezel) [reply code/modules/antagonists/cult/cult_items.dm]: I thought I deleted it, let me check

Comment (Sparkezel) [reply code/modules/antagonists/cult/cult_items.dm]: I know I can get rid of angle very easly, not sure about the first 2 lines.

**Comment by LemonInTheDark** [comment]
Can you explain exactly what was going wrong here?

**Comment by Sparkezel** [comment]
The problem with old Blood beam targetting, was that it used a command from byond that updated the angle only when you clicked and dragged, from what I understood what AnturK has told me.

**Comment by Angustmeta** [comment]
Finally closes #44413, one of my oldest issue reports. This is a moment ro celebrate.

**Comment by ATH1909** [comment]
> Last I checked it still delt 20 damage to cultists, however that was on a catatonic mob so I am not sure.

Oh no.

**Pull request merged**